From 1b4be3a6d4f76eaa14f31b53c4a715fe5684f201 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:14:00 +0800
Subject: [PATCH 0165/5242] MLK-11123-31 ARM: dts: disable otg features for
 imx7d-sdb board

commit  0e7b9c3d49a2d504d97f7e5dacca31710351ab50 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable all otg features for otg1 port of imx7d-sdb board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit 3c59511b7eac99da8c9bd373b82ddcbf61d1ef8a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index ff574e9..09e4166 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -407,6 +407,9 @@
 
 &usbotg1 {
 	vbus-supply = <&reg_usb_otg1_vbus>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.9.5

